### DISCUSSION TITLE
Addon idea: Add an option to delete project in editor or project page (if it is yours)

### DISCUSSION THREAD

**Discussion by leofleeo** [ScratchAddons/ScratchAddons#3857] [Ideas]
This would be very useful so you do not have to go digging in the my stuff page (but there is already my stuff search but whatever it would be a pretty good improvement).

**Comment by WorldLanguages** [comment]
Probably rejected. My stuff sorts by last accessed/modified already. And deleting is scary.

**Comment by leofleeo** [comment]
It would be useful for deleting really old trash projects you made when you were 5
